### PR TITLE
Add first objects (Season, User)

### PR DESCRIPTION
**Pull request by xWorkerBeex**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "axios": "^0.19.2",
```
Comment (lsocrate) [inline client/package.json]: Cool library

File: client/src/views/SeasonView.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+export class SeasonView extends React.Component<{}, SeasonViewState> {
+    constructor(props: any) {
+        super(props)
+        this.state = {seasons: []}
+    }
+
+    componentDidMount() {
+        axios.get("http://localhost:8080/seasons").then(resp => {
```
Comment (lsocrate) [inline client/src/views/SeasonView.tsx]: We could abstract the API calls later to a central point, to make it easier to manage the authorization tokens, and the hosts for dev and production, etc. No need to sweat it now, but a thing to think for the next steps

File: src/clients/user.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import {DiscordUser} from "./discord"
+import {Clan} from "../constants/clan"
+
+export interface User {
+    discordUser: DiscordUser
+    preferredClan?: Clan
+    elo?: number
```
Comment (lsocrate) [inline src/clients/user.ts]: 👍

File: src/season/season.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import { SeasonStatus } from './seasonStatus';
+
+export class Season {
+    id: String
+    name: String
+    status: SeasonStatus
+
+    constructor(id: String, name: String) {
```
Comment (lsocrate) [inline src/season/season.ts]: When the constructor is used just to assign properties we can take a shortcut and do it like:

```ts
export class Season {
  constructor(public id: String, public name: String, public status = SeasonStatus.Upcoming) {}
}
```

The received properties get directly assigned to the instance 🙂

File: client/package.json (hunk 1 above)
Comment (xWorkerBeex) [reply client/package.json]: It's what I used in my bachelor thesis. It was recommended by a co-worker of mine who is working mostly on React or Vue projects :)

File: client/src/views/SeasonView.tsx (hunk 2 above)
Comment (xWorkerBeex) [reply client/src/views/SeasonView.tsx]: Yeah, that makes sense :) I wanted to just get it to work, we can move that around later :)

File: src/season/season.ts (hunk 4 above)
Comment (xWorkerBeex) [reply src/season/season.ts]: Awesome, the more you know!

**Pull request merged**